FeatureToggle.Domain/Entity/FeatureManagementSchema/User.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Identity;
+
+namespace FeatureToggle.Domain.Entity.FeatureManagementSchema
+{
+    public class User : IdentityUser
+    {
+        public string Name { get;private set; }
+        public bool IsAdmin { get; private set; }
+        //public List<Log> Logs { get; private set; }
```

Comment:
Remove all commented codes and empty spaces. CTRL + K + D to format the code and use CTRL + R + G to remove unused imports

Reply:
corrected